src/game/animation/startBallAnimation.ts

```diff
@@ -30,6 +34,11 @@ export function startBallAnimation(
   function updateBall(ball: Ball): void {
     updateBallVelocity(ball)
     updateBallPositionBy(ball, currentBallVelocity)
+    // To prevent the ball from falling into a loop as much as possible
+    const fluctuation = barSetting.width * 0.4 - Math.random() * 5;
+    if (startOptions.demo) {
+      moveBarTo(bar, getBallCenterPosition(ball).x - fluctuation)
+    }
```

Comment:
really cool!! I wouldn't notice the ball could fall into a loop. thanks!! :)
Just a minor suggestion, this might be a bit clearer?
```suggestion
    // To prevent the ball from falling into a loop as much as possible
    if (startOptions.demo) {
      const fluctuation = barSetting.width * 0.4 - Math.random() * 5;
      moveBarTo(bar, getBallCenterPosition(ball).x - fluctuation)
    }
```

Reply:
Thank you! Modified as you suggested!